homeassistant/components/seventeentrack/__init__.py

```diff
@@ -1 +1,68 @@
 """The seventeentrack component."""
+from py17track import Client as SeventeenTrackClient
+from py17track.errors import SeventeenTrackError
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_SHOW_ARCHIVED, CONF_SHOW_DELIVERED, DOMAIN
+
+PLATFORMS = [Platform.SENSOR]
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): str,
+                vol.Required(CONF_PASSWORD): str,
+                vol.Optional(CONF_SHOW_ARCHIVED, default=False): bool,
+                vol.Optional(CONF_SHOW_DELIVERED, default=False): bool,
+            },
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the 17Track component."""
+    hass.data.setdefault(DOMAIN, {})
+
+    if hass.config_entries.async_entries(DOMAIN):
+        return True
```

Comment:
We should create repair issues instead to point users to the fact that they need to remove the YAML. Please checkout integrations like `lutron` and `suez_water` for this.

Reply:
Done